cmd/apps/confluentkafka.go

```diff
@@ -0,0 +1,140 @@
+package apps
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallConfluentPlatformKafka() *cobra.Command {
+	kafka := &cobra.Command{
+		Use:   "kafka",
+		Short: "Install Confluent Platform Kafka",
+		Long: `This will install Kafka provided by the Confluent Platform by using the following official Helm chart:
+        https://github.com/confluentinc/cp-helm-charts`,
+		Example:      "",
+		SilenceUsage: true,
+	}
+
+	kafka.Flags().Bool("enable-zookeeper", true, "enable Zookeeper")
+	kafka.Flags().Int("zookeeper-server-count", 3, "server count of Zookeeper")
+
+	kafka.Flags().Bool("enable-kafka", true, "enable Kafka")
+	kafka.Flags().Int("kafka-broker-count", 3, "broker count of Kafka")
+
+	kafka.Flags().Bool("enable-schema-registry", true, "enable Schema Registry")
```

Comment:
Rather than `--enable-ksql-server` can we just have `-ksql-server` for shorter flags?

Also turn everything off other than Kafka and Zookeeper.

Only 1 replica, not 3, see the notes in the issue which mention the same - #446